src/libs/actions/Report.ts

```diff
@@ -2148,10 +2154,7 @@ function openReportFromDeepLink(url: string, isAuthenticated: boolean) {
         Session.waitForUserSignIn().then(() => {
             Navigation.waitForProtectedRoutes().then(() => {
                 const route = ReportUtils.getRouteFromLink(url);
-                if (route === ROUTES.CONCIERGE) {
-                    navigateToConciergeChat(true);
-                    return;
-                }
```

Comment:
This prevents us to deep-link to ConciergePage when logged out.

Reply:
Is there any issue caused by this?